spec/unit/network/formats_spec.rb

```diff
@@ -130,6 +130,24 @@ def to_pson(*args)
     end
   end
 
+  describe "binary" do
+    before do
+      @binary = Puppet::Network::FormatHandler.format(:binary)
+    end
+
+    it "should have its mimetype set to application/octet-stream" do
+      @binary.mime.should == "application/octet-stream"
+    end
```

Comment:
Can we switch these to rspec 3 style, e.g.
```
expect(@binary.mime).to eq("application/octet-stream")
```

And similarly for the spec changes below.